assets/prototype/domain/eventEditor/data/types.ts

```diff
@@ -58,6 +58,7 @@ export interface Price extends Entity {
 	isTax: boolean;
 	order: number | string;
 	overrides: string;
+	priceType: string;
```

Comment:
This is the wrong assumption, it should be `PriceType` or we need to come up with an alternative:

GQL Schema for `Price`
![image](https://user-images.githubusercontent.com/18226415/72724495-75cb7000-3ba9-11ea-81e3-d5e3377738f8.png)